crates/core/app/tests/mock_consensus_staking.rs

```diff
@@ -186,6 +186,95 @@ async fn mock_consensus_can_define_and_delegate_to_a_validator() -> anyhow::Resu
         );
     }
 
+    // Now, create a transaction that delegates to the new validator.
+    let plan = {
+        use {
+            penumbra_num::Amount,
+            penumbra_sct::component::clock::EpochRead,
+            penumbra_stake::Delegate,
+            penumbra_transaction::{ActionPlan, TransactionParameters, TransactionPlan},
+        };
+        let action = ActionPlan::Delegate(Delegate {
+            validator_identity: new_validator_id,
+            epoch_index: storage.latest_snapshot().get_current_epoch().await?.index,
+            unbonded_amount: Amount::zero(), // TODO: use a non-zero value here.
```

Comment:
This is probably why the binding sig fails, when you delegate you "burn" staking tokens (i.e. unbonded amount) using a rate (validator exchange) that lets you mint delegation tokens (bound to a specific validator).

This will be helpful to visualize it:

```rust
    /// Return the balance resulting from issuing delegation tokens from staking tokens.
    pub fn balance(&self) -> Balance {
        let stake = Balance::from(Value {
            amount: self.unbonded_amount,
            asset_id: STAKING_TOKEN_ASSET_ID.clone(),
        });

        let delegation = Balance::from(Value {
            amount: self.delegation_amount,
            asset_id: DelegationToken::new(self.validator_identity.clone()).id(),
        });

        // We produce the delegation tokens and consume the staking tokens.
        delegation - stake
    }
}
```